great_expectations/core/expectation_diagnostics/expectation_diagnostics.py

```diff
@@ -18,7 +21,21 @@
     ExpectationTestDiagnostics,
 )
 from great_expectations.core.util import convert_to_json_serializable
+from great_expectations.expectations.registry import get_expectation_impl
 from great_expectations.types import SerializableDictDot
+from great_expectations.util import camel_to_snake, lint_code
+
+try:
+    import black
+except ImportError:
+    black = None
+
+try:
+    import isort
+except ImportError:
+    isort = None
+
+rx_expectation_instance_repr = re.compile(r"<.*\.([^\.]*) object at .*")
```

Comment:
If this regex is only used in the single method, could we perhaps do this within the function? I'd rather clean up the global namespace if at all possible, even if it means being slightly less performant.